.github/workflows/acctest.yml

```diff
@@ -119,10 +119,9 @@ jobs:
         METAL_AUTH_TOKEN: ${{ secrets.METAL_AUTH_TOKEN }}
         SWEEP: "all" #Flag required to define the regions that the sweeper is to be ran in
         SWEEP_ALLOW_FAILURES: "true" #Enable to allow Sweeper Tests to continue after failures
-        SWEEP_DIR: "./equinix"
       run: |
         # Added sweep-run to filter Metal test
-        go test ${SWEEP_DIR} -v -timeout 180m -sweep=${SWEEP} -sweep-allow-failures=${SWEEP_ALLOW_FAILURES} -sweep-run=$(grep -o 'AddTestSweepers("[^"]*"' equinix/resource_metal_* |cut -d '"' -f2 | paste -s -d, -)
+        go test $(go list ./... | grep 'internal/sweep\|equinix/equinix') -v -timeout 180m -sweep=${SWEEP} -sweep-allow-failures=${SWEEP_ALLOW_FAILURES} -sweep-run=$(grep -or 'AddTestSweepers("[^"]*"' | grep "_metal_" |cut -d '"' -f2 | paste -s -d, -)
```

Comment:
This change has to be run via manual trigger because the pull_request_target trigger uses the workflow from the base branch (in this case, `main`).  Test run link pending.

Reply:
Taking the opportunity to update the sweeper to handle locked devices.  Manual run in progress here: https://github.com/equinix/terraform-provider-equinix/actions/runs/8071102002/job/22049993375